src/server/worker/api_server.go

```diff
@@ -108,13 +108,16 @@ type taggedLogger struct {
 	buffer       bytes.Buffer
 	putObjClient pfs.ObjectAPI_PutObjectClient
 	objSize      int64
+	msgCh        chan string
+	eg           errgroup.Group
 }
 
 func (a *APIServer) getTaggedLogger(ctx context.Context, req *ProcessRequest) (*taggedLogger, error) {
 	result := &taggedLogger{
 		template:  a.logMsgTemplate, // Copy struct
 		stderrLog: log.Logger{},
 		marshaler: &jsonpb.Marshaler{},
+		msgCh:     make(chan string, 10),
```

Comment:
I'd be down to make it a bit bigger, maybe `25` anything larger than that really seems like overkill to me though